studygroups/views/api.py

```diff
@@ -981,13 +981,16 @@ def get(self, request, **kwargs):
 
 def serialize_team_membership(tm):
     role_label = dict(TeamMembership.ROLES)[tm.role]
+    email_validated = hasattr(tm.user, 'profile') and tm.user.profile.email_confirmed_at is not None
+    email_confirmed_at = tm.user.profile.email_confirmed_at.strftime("%-d %B %Y") if email_validated else "--"
```

Comment:
Any reason you're using `Profile.email_confirmed_at` rather than `User.date_joined`